R/make_stancode.R

```diff
@@ -307,6 +307,12 @@ make_stancode <- function(formula, data, family = gaussian(),
   if (parse) {
     scode <- parse_model(scode, backend, silent = silent)
   }
+  if (backend == "cmdstanr") {
+    if (cmdstanr::cmdstan_version() >= "2.29.0") {
+      tmp_file <- cmdstanr::write_stan_file(scode)
+      scode <- .canonicalize_stan_model(tmp_file, overwrite_file = FALSE)
```

Comment:
We didn't discuss this in the issue, but I added this for make_stancode as well, so that the output code also doesn't produce warnings if cmdstanr is used.